src/www/app/app.ts

```diff
@@ -353,20 +351,18 @@ export class App {
     }
 
     const server = this.getServerByServerId(serverId);
-    const card = this.getCardByServerId(serverId);
-
     console.log(`connecting to server ${serverId}`);
 
-    card.state = 'CONNECTING';
+    this.updateUiServer(serverId, {state: ServerConnectionState.CONNECTING});
```

Comment:
I would personally use `ServerUI` rather than `UiServer` - the latter to me reads as a server that serves the UI (in the SSR sense) while the former more is more clearly the UI for the server

Reply:
Renamed to `updateServerUi`.

I meant "the Server concept in the UI", versus the "Server in the Domain Model" (which we just call `Server`). In part because we just call them "servers" in the UI code.

Not that within the function `serverUi` doesn't make as much sense (it's not the server UI), so I didn't rename it. I could call it `serverModel`, but it's ambiguous with the domain model and I want to make it clear it's a UI concept.

Reply:
Maybe `updateServerCard` because that's consistent with `ServerCardModel`?

Reply:
Renamed